docs/guide/protocols/absolutesolvation.rst

```diff
@@ -1,4 +1,106 @@
+The absolute solvation free energy is then obtained via summation of free energy differences along the thermodynamic cycle.
+
+.. figure:: img/ahfe_thermocycle.png
+   :scale: 100%
+
+   Thermodynamic cycle for the absolute solvation free energy protocol.
+
+Scientific Details
+------------------
+
+Partial annhilation scheme
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the :class:`.AbsoluteSolvationProtocol` the coulombic interactions of the molecule are fully turned off (annihilated),
+while the Lennard-Jones interactions are decoupled, meaning the intermolecular interactions turned off, keeping the intramolecular Lennard-Jones interactions.
+
+The lambda schedule
+~~~~~~~~~~~~~~~~~~~
+
+Molecular interactions are turned off during an alchemical path using a discrete set of lambda windows. The electrostatic interactions are turned off first, followed by the decoupling of the van-der-Waals interactions. A soft-core potential is applied to the Lennard-Jones potential to avoid instablilites in intermediate lambda windows. 
+
+Simulation overview
+~~~~~~~~~~~~~~~~~~~
+
+The :class:`.ProtocolDAG` of the :class:`.AbsoluteSolvationProtocol` contains both the units from the vacuum and from the solvent transformations.
+This means that both legs of the thermodynamic cycle are constructured and run concurrently in the same :class:`.ProtocolDAG`. This is different from the :class:`RelativeHybridTopologyProtocol` where the :class:`.ProtocolDAG` only runs a single leg of a thermodynamic cycle.
+If multiple ``protocol_repeats`` are run (default: ``protocol_repeats=3``), the :class:`.ProtocolDAG` contains multiple units of both vacuum and solvent transformations.
+
+Simulation steps
+""""""""""""""""
+
+Each Protocol simulation Unit (whether vacuum or solvent) carries out the following steps:
+
+1. Parameterize the system using `OpenMMForceFields <https://github.com/openmm/openmmforcefields>`_ and `Open Force Field <https://github.com/openforcefield/openff-forcefields>`_.
+2. Equilibrate the fully interacting system using a short MD simulation using the same approach as the :class:`.PlainMDProtocol` (in the solvent leg this will include rounds of NVT and NPT equilibration)
+3. Create an alchemical system
+4. Minimize the alchemical system
+5. Equilibrate and production simulate the alchemical system using the chosen multistate sampling method (under NPT conditions if solvent is present).
+6. Analyze results for the transformation
+
+Note: three different types of multistate sampling (i.e. replica swapping between lambda states) methods can be chosen; HREX, SAMS, and independent (no lambda swaps attempted). By default the HREX approach is selected, this can be altered using ``solvent_simulation_settings.sampler_method`` or ``vacuum_simulation_settings.sampler_method`` (default: ``repex``).
+
+Simulation details
+""""""""""""""""""
+
+The protocol applies a 
```

Comment:
We could keep this section a bit loose and do some bullet points with a brief single sentence "here are some details of how the simulation is carried out which are not detailed in the Protocol Settings (reference to the settings)"?

This is probably a section we will eventually need to copy paste a bunch of places. Assuming we were writing a paper, are there any other details we would include? I'm not sure I can think of any :/